_data/payments.yml

```diff
@@ -125,8 +125,9 @@ websites:
       img: squareup.png
       tfa: Yes
       sms: Yes
+      software: Yes
       exceptions:
-        text: "Currently only US merchants, protects only Dashboard access"
+        text: "Outside US/CA, only Google Authenticator (or any TOTP client) are supported"
```

Comment:
Perhaps rephrase it that SMS is supported only in us/canada? Since that's the limitation, not totp.